scalalib/src/mill/scalalib/MiscModule.scala

```diff
@@ -65,13 +65,13 @@ trait MavenModule extends JavaModule{outer =>
   )
   override def resources = T.sources{ millSourcePath / 'src / 'main / 'resources }
   trait Tests extends super.Tests with MavenTests {
-    override def millSourcePath = outer.millSourcePath
+    override def millSourcePath = outer.millSourcePath / 'src / 'test
```

Comment:
I thought that we can make `parentModule/src/test` as a root for test module. @lihaoyi  Don't you mind that?

Reply:
Or maybe we can do special handling of base path in GenIdea itself, cause this change may break current builds, when you rely on path of your test module. This happened in Play Json build.